services/mentor-api/Makefile

```diff
@@ -53,7 +54,21 @@ docker-run:
 			--name $(DOCKER_CONTAINER) \
 			--shm-size 8G \
 			-p 5000:5000 \
-		$(DOCKER_IMAGE) 
+		$(DOCKER_IMAGE)
+
+
+.PHONY: docker-run-mount-checkpoint
+docker-run-mount-checkpoint:
```

Comment:
I'd prefer to not have a CHECKPOINT global variable here. Instead can this rule be

```
docker-run-checkpoint/%
```

?